subgraph_extractor/cli.py

```diff
@@ -216,6 +217,61 @@ def write_config(config, root_output_location):
             yaml.dump(config, f_out)
 
 
+def remove_protocol(path):
```

Comment:
Leaky abstractions around cloud vs local files.

Reply:
At first sight, `remove_protocol` don't seem like a good function name. `remove_cloud_prefix` is more direct.

Reply:
renamed, I like that change